src/Nethermind/Nethermind.Network/Discovery/DiscoveryManager.cs

```diff
@@ -221,25 +220,6 @@ private bool ValidatePingAddress(PingMessage message)
                 return false;
             }
 
-            // port will be different as we dynamically open ports for each socket connection
```

Comment:
these were kind of comments - maybe we should just add comments instead of code?